.github/workflows/install-prod.yml

```diff
@@ -0,0 +1,70 @@
+name: Prod Deploy (install / demo / nocturnal / raindrop scripts)
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+
+env: 
+  ENVIRONMENT: prod
+  S3_BUCKET: ${{ secrets.BUCKET_NAME }}
+  AWS_REGION: ${{ secrets.BUCKET_REGION }}
+
+jobs:
+  build_ubuntu:
+
+    runs-on: ubuntu-latest
+    environment: prod
+
+    name: Prod Deploy (install / demo / nocturnal / raindrop scripts)
+
+    steps:
+      - name: Check out code. 
+        uses: actions/checkout@v3
+      
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-region: ${{ env.AWS_REGION }}
+          aws-access-key-id: ${{ secrets.STAGING_S3_DEPLOYMENT_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.STAGING_S3_DEPLOYMENT_SECRET_KEY }}
+
+      - name: Copy files to the s3 with the AWS CLI
```

Comment:
shouldn't presto be on this list too?

where are the compiled ones: moonlight and hades?

Reply:
They are going to be directory by directory, this is the only one I was done playing with. The rest are coming.

Reply:
kk

Reply:
so is this a standalone PR or a WIP?

Reply:
stand alone. should be good to go.